src/lib/orionld/serviceRoutines/orionldPostEntities.cpp

```diff
@@ -220,13 +220,13 @@ static bool payloadCheck
   //
   if (idNodeP == NULL)
   {
-    orionldErrorResponseCreate(ciP, OrionldBadRequestData, "No 'id' of the entity", "The 'id' field is mandatory", OrionldDetailsString);
+    orionldErrorResponseCreate(ciP, OrionldBadRequestData, "Entity id is missing", "The 'id' field is mandatory", OrionldDetailsString);
     return false;
   }
 
   if (typeNodeP == NULL)
   {
-    orionldErrorResponseCreate(ciP, OrionldBadRequestData, "No type of the entity", "The type field is mandatory", OrionldDetailsString);
+    orionldErrorResponseCreate(ciP, OrionldBadRequestData, "Entity type missing", "The type field is mandatory", OrionldDetailsString);
```

Comment:
For consistency (with 'Entity id is missing'), add 'is'

Reply:
Fixed: 17ee16d13